dbt_project_delete_test.go

```diff
@@ -0,0 +1,19 @@
+package fivetran_test
+
+import (
+	"context"
+	"strings"
+	"testing"
+)
+
+func TestNewProjectDeleteE2E(t *testing.T) {
+	groupId := CreateProject(t)
+	deleted, err := Client.NewGroupDelete().GroupID(groupId).Do(context.Background())
+	if err != nil {
+		t.Logf("%+v\n", deleted)
+		t.Error(err)
+	}
+	AssertEqual(t, deleted.Code, "Success")
+	AssertNotEmpty(t, deleted.Message)
+	AssertEqual(t, strings.Contains(deleted.Message, groupId), true)
```

Comment:
What is the logic here?
You're creating a dbt project, and then deleting a group? What the point?
And tests are failing because you now have existing project you didn't deleted.

Before enabling e2e test it's better to test it on a local environment.